solutions/recursion/binomial-coefficient.js

```diff
@@ -0,0 +1,35 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+
+
+function binomialCoeff(n, r) {
```

Comment:
@DevratAgarwal  function name can be binomialCoefficient if it is okay

Reply:
@satya-9 that's template of the code should i change it ?

Reply:
@DevratAgarwal  Template I changed. Please check once again

Reply:
@satya-9 it is not visible